### DISCUSSION TITLE
Where is the dark mode option？

### DISCUSSION THREAD

**Discussion by NebulaTechs** [filecxx/FileCentipede#639] [Q&A]
Where is the dark mode option？

**Comment by filecxx** [comment]
Probably in the future.

**Reply by IkelAtomig** [reply]
I think you should let the system handle and as your are using Qt. I don't think it would be harder like using a module. Sorry, I understand it would be a lot of work to do though. I am just saying.

**Comment by aTomicTurnip** [comment]
Would be a nice addition🐦‍⬛
